### PR TITLE
fix setup.md

### PR DESCRIPTION
**Pull request by dwSun**
> /kind documentation

### CODE REVIEW

File: docs/setup/setup.md [hunk 1]
```diff
@@ -117,26 +118,44 @@ The cert/key will be generated in the `/etc/kubeedge/ca` and `/etc/kubeedge/cert
...
++ Run cloudcore
 
     ```shell
     cd ~/cmd/
-    nohup ./cloudcore & 
+    nohup ./cloudcore &
     ```
-    
-#### Deploy the edge node 
+    **Note:** Please run cloudcore using the users who have root permission.
```
[the remark this answers is not in the document]
Comment (dwSun) [reply docs/setup/setup.md]: why not.

File: build/tools/kecloud.service [hunk 2]
```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=kecloud.service
```
[the remark this answers is not in the document]
Comment (dwSun) [reply build/tools/kecloud.service]: agree

File: docs/setup/setup.md [hunk 3]
```diff
@@ -314,13 +324,13 @@ cd $GOPATH/src/github.com/kubeedge/kubeedge
     ```shell
     sudo ln build/tools/edge.service /etc/systemd/system/edge.service
     sudo systemctl daemon-reload
-    sudo systemctl start edgecore
+    sudo systemctl start edge
```
[the remark this answers is not in the document]
Comment (dwSun) [reply docs/setup/setup.md]: I personally prefer **edgecore** and **cloudcore** too.

File: docs/setup/setup.md [hunk 4]
```diff
@@ -16,7 +16,8 @@ KubeEdge is composed  of cloud and edge parts. It is built upon Kubernetes and p
 
 + [Creating kubernetes cluster with kubeadm on cloud side](<https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/>)
 
-+ **Go** The minimum required go version is 1.12. You can install this version by using [this website.](https://golang.org/dl/) 
++ **Go** The minimum required go version is 1.12. You can install this version by using [this website.](https://golang.org/dl/)
++ [Install mosquitto on every edge side](https://mosquitto.org/)
```
[the remark this answers is not in the document]
Comment (dwSun) [reply docs/setup/setup.md]: better leave it here.

File: docs/setup/setup.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (fisherxu) [reply docs/setup/setup.md]: Cloudcore just load the `certs and config files` from host, other users with this permission are also ok? :)

[the remark this answers is not in the document]
Comment (kadisi) [reply docs/setup/setup.md]: agree,  cloudcore no need root permission

[the remark this answers is not in the document]
Comment (dwSun) [reply docs/setup/setup.md]: Got it.

**Comment by kadisi** [comment]
please squash your commit, then lgtm

**Comment by fisherxu** [comment]
/lgtm 
Thanks! @dwSun

**Pull request merged**